app/src/common/docs/BaseDocsBlock.module.scss

```diff
@@ -22,6 +25,10 @@
 .width-wrapper {
     padding: 0 60px 60px;
     max-width: 960px;
+
+    h1, h2, h3, h4, h5, h6 {
+        color: var(--uui-text-dark);
+    }
```

Comment:
There is no need to set the heading color here, it is already set in the typography